### PR TITLE
sum_for_loop

### PR DESCRIPTION
**Pull request by Nibedita-naik**
program to print sum of first n natural numbers by using for loop.

### CODE REVIEW

File: 1.Practicals_C/Miscc/profit or loss.c [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
+int main()
+{
+    float cp, sp, p, l;
+    printf("\nEnter cost price and selling price:");
+    scanf("%f %f", &cp, &sp);
+    p = sp - cp;
```
Comment (sarvesh-0) [inline 1.Practicals_C/Miscc/profit or loss.c:7]: This logic can be improved

Comment (sarvesh-0) [reply 1.Practicals_C/Miscc/profit or loss.c:7]: You can have something like,
```
if (p>0)
        printf("Profit: %f \n", p);
else if (p< 0)
        printf("Loss: %f \n", -p);
else
 printf("No profit, No loss ")
```

This way, computing l wouldn't be required. And, the logic would be computationally efficient.

File: 1.Practicals_C/Miscc/loss_or_profit.c [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#include <stdio.h>
+int main()
+{
+    float cp, sp, p, l;
```
Comment (sarvesh-0) [inline 1.Practicals_C/Miscc/loss_or_profit.c:4]: No need to declare variable 'l' over here.

File: 1.Practicals_C/Decision_Making/Looping/Implementation/sum.c [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
```
Comment (sarvesh-0) [inline 1.Practicals_C/Decision_Making/Looping/Implementation/sum.c:1]: Write in one line or two that what's the problem statement this program is intending to solve.

Comment (Nibedita-naik) [reply 1.Practicals_C/Decision_Making/Looping/Implementation/sum.c:1]: ok sir !!

**Review by sarvesh-0 — APPROVED** [review]

**Pull request merged**